### DISCUSSION TITLE
Using websocket with wss when using https

### DISCUSSION THREAD

**Discussion by liberty-rising** [idom-team/idom#714] [Problem]
Hey I've been working on deploying the timeflow app and we are currently working on getting it to use https. However, upon what looks like a successful deployment, we are running into a web socket issue.

Here's a link to the site: [timeflow app](https://demo-timeflow.dyvenia.com)

If you look at the developer console, the issue appears to be a lack of support for wss.

`Mixed Content: The page at 'https://demo-timeflow.dyvenia.com/client/index.html' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://demo-timeflow.dyvenia.com/stream'. This request has been blocked; this endpoint must be available over WSS.`

Would it be possible to create support for this protocol?

Cheers!

**Comment by acivitillo** [comment]
Could this be a typo bug [here](https://github.com/idom-team/idom/blob/main/src/client/packages/idom-app-react/src/index.js)

```javascript
export function mount(mountPoint) {
  const serverInfo = new LayoutServerInfo({
    host: document.location.hostname,
    port: document.location.port,
    path: "../",
    query: queryParams.user.toString(),
    secture: document.location.protocol == "https",
  });
```

`secture` instead of `secure`

**Reply by rmorshea** [reply]
Yup. I think that's the cause. Can you make a PR to correct that? I can release this later today.

**Reply by acivitillo** [reply]
@liberty-rising sent a PR that should fix this https://github.com/idom-team/idom/pull/716

**Comment by acivitillo** [comment]
sure, we’ll send a PR for this
